book/browsersandweb.md

```diff
@@ -266,48 +258,44 @@ fascinating and symbiotic way with the huge number of web pages deployed today.
 The web in history
 ==================
 
-The public Internet and the Web co-evolved, and in fact many peoples’ first
-experiences of the Internet from the 1990s onward were more or less
-experiences with the web. However, it’s important to distinguish between them,
-since the Internet and the web are in fact not synonymous.
-
-In the early days, the similarity between the _physical structure_ of the
-web---where the web servers were---and the _names_ of the [websites][website]
-was very strong. The Internet was a world wide network of computers, those
-computers had domain names, many of them ran web servers, and those servers
-stored and provided web pages to browsers that asked for them. In this sense, the
-Internet and the web really were closely related at that time. However, there is
-of course nothing inherent about this: nothing forces you to host your own web
-server on your home computer and Internet connection,[^self-hosted] and the same
-goes for a university or corporation. Likewise, there is nothing requiring
-everyone to have their own website rather than a social networking account.
-These days, almost everyone uses a virtual machine or service purchased from one
-kind of cloud computing service or another to run their websites, regardless of
-how small or large, and there are many products available that can easily
-publish your web content on your behalf on various social networking platforms.
-
-[website]: https://en.wikipedia.org/wiki/Website
-
-[^self-hosted]: People actually did this! And when their website became
-popular, it often ran out of bandwidth or computing power and became
-inaccessible.
-
-This same *virtualization* concept also applies to the implementation of web
-pages themselves. While it’s still possible to write HTML by hand, few of the
-most popular web pages' HTML payloads literally exist on a hard drive somewhere.
-Instead, their component pieces and dependent databases exist, and the final
-product is dynamically assembled on the fly by complex build and
-“rendering”[^server-side-rendering] systems and sent over the Internet on-demand
-to your browser. The reason for this is that their contents themselves are
-dynamic---composed of data from news, blog posts, inbox contents,
-advertisements, and algorithms adjusting to your particular tastes.
+It might seem natural to use a web browser to browser the Web over the Internet,
+and many peoples’ experience of the Internet from the 1990s onward is like this.
+This can make the Web seem already-built, with your place being to browse it.
+But the Web is neither simple nor obvious. It is the result of experiments and
```

Comment:
Lowercase "web" throughout. Or not - should the Web when used as a proper noun be capitalized? And when used as an adjective, lowercase?